lib/chef/resource/helpers/cron.rb

```diff
@@ -0,0 +1,19 @@
+class Chef
+  module ResourceHelpers
+    module Cron
+      WEEKDAYS = { sunday: '0', monday: '1', tuesday: '2', wednesday: '3', thursday: '4', friday: '5', saturday: '6',
+                    sun: '0', mon: '1', tue: '2', wed: '3', thu: '4', fri: '5', sat: '6'
+                  }
```

Comment:
nit: a good rule of thumb is - either both braces line by themself, or none should be. Personally I'd go:

```
      WEEKDAYS = {
        sunday: '0', monday: '1', tuesday: '2', wednesday: '3', thursday: '4', friday: '5', saturday: '6',
        sun: '0', mon: '1', tue: '2', wed: '3', thu: '4', fri: '5', sat: '6'
      }
```

Reply:
@jaymzh Agreed, I updated the same.